src/BenchmarkDotNet/Columns/FirstCallColumn.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using BenchmarkDotNet.Extensions;
+using BenchmarkDotNet.Reports;
+using BenchmarkDotNet.Running;
+
+namespace BenchmarkDotNet.Columns
+{
+    public class FirstCallColumn : IColumn
+    {
+        public string Id => nameof(FirstCallColumn);
+
+        public string ColumnName => "First Call";
+
+        public bool AlwaysShow => true;
+
+        public ColumnCategory Category => ColumnCategory.Custom;
+
+        public int PriorityInCategory => -1;
+
+        public bool IsNumeric => true;
+
+        public UnitType UnitType => UnitType.Time;
+
+        public string Legend => "Excution time of the first call (Jitting included)";
+
+        public string GetValue(Summary summary, BenchmarkCase benchmarkCase)
+        {
+            if (!summary.HasReport(benchmarkCase) || !summary[benchmarkCase].Success || !summary[benchmarkCase].AllMeasurements.Any(IsFirstCall))
+                return "-";
+
+            var measurement = summary[benchmarkCase].AllMeasurements.Single(IsFirstCall);
```

Comment:
Theoretically, the Engine may perform several JittingWorkload iterations that will lead to an exception during summary table building. It will be better to select all `IsFirstCall` measurements, sort them by IterationStage, then by `LaunchIndex`, and take the first measurement. If you think that a situation with several "FirstCall" measurements is an error that should be reported, please use analyzers instead; don't throw exceptions in `IColumn.GetValue`.

Reply:
good point, I've added sorting